client/js/fun.js

```diff
@@ -0,0 +1,9 @@
+var fun = function(input) {
```

Comment:
Down with "var"! Up with either "const" or "let". "var" went the way of the birds when ES6 came out (although you'll still see it in outdated code).